client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -225,6 +251,32 @@ const MetadataEditor = createClass({
 		</div>;
 	},
 
+	renderLanguageDropdown : function(){
+		const langCodes = ['en', 'de', 'fr', 'ja', 'es', 'it'];
+		const listLanguages = ()=>{
+			return _.map(langCodes, (code)=>{
+				const languageNames = new Intl.DisplayNames([code], { type: 'language' });
+				return <option value={`${code}`}>{`${languageNames.of(code)}`}</option>;
+			});
+		};
+
+
+
+		return <div className='field language'>
+			<label>language</label>
+			<input type='text' className='value'
+				value={this.props.metadata.lang}
+				onChange={(e)=>this.handleFieldChange('lang', e)}
+				list='languageList'
```

Comment:
Anytime an `input` has a `list` attribute, it is usually a good idea to turn off `autocomplete` .. otherwise the UI for autocomplete can also appear, and the input now has two drop-down lists of suggestions. It still works, but it looks very silly.

Reply:
committed and pushed